src/Symfony/Component/Notifier/Bridge/Lox24/Lox24Options.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Notifier\Bridge\Lox24;
+
+use InvalidArgumentException;
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author Andrei Lebedev <andrew.lebedev@gmail.com>
+ */
+final class Lox24Options implements MessageOptionsInterface
+{
+    public function __construct(
+        private array $options = [],
+    ) {
+    }
+
+    public function toArray(): array
+    {
+        return $this->options;
+    }
+
+    public function getRecipientId(): ?string
+    {
+        return null;
+    }
+
+
+    /**
+     * Unix timestamp of SMS the delivery time. If 0 or not set, the message will be sent immediately.
+     */
+    public function deliveryAt(int $deliveryAt): self
+    {
+        $this->options['delivery_at'] = max($deliveryAt, 0);
+
+        return $this;
+    }
+
+
+    /**
+     * The language of the voice message.
+     * If not set, the automatic language detection by message text will be used.
+     */
+    public function voiceLanguage(?string $language): self
+    {
+        if($language) {
+            $language = strtolower($language);
+        }
+
+        if ($language && !in_array($language, Lox24Transport::ALLOWED_VOICE_LANGUAGES, true)) {
+            throw new InvalidArgumentException(
```

Comment:
Please on one line, it avoids merge conflicts in the future

Reply:
Should i split condition to two lines?

Reply:
No

Reply:
done